Ginger/GingerCore/Actions/ActLaunchJavaWSApplication.cs

```diff
@@ -88,6 +88,7 @@ public new static partial class Fields
             public static string WaitForWindowWhenDoingLaunch = "WaitForWindowWhenDoingLaunch";   //flag to determine if to wait for java window with required title when launching java application
             public static string WaitForWindowTitle = "WaitForWindowTitle"; //the title of the Java application to wait for
             public static string WaitForWindowTitleMaxTime = "WaitForWindowTitleMaxTime";    //the max time in seconds to wait for the window to load
+            public static string AttachAgentProcessSyncTime = "AttachAgentProcessSyncTime";    //the max time in seconds to wait for the process holding the mutex
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'AttachAgentProcessSyncTime' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5169785)